python/README.md

```diff
@@ -30,6 +30,20 @@ to request, import, and manage certificates.
 * Retrieve certificate data.
 * Add custom tags to certificates.
 
+### [Amazon CloudWatch custom metrics and alarms example](https://github.com/awsdocs/aws-doc-sdk-examples/tree/master/python/example_code/cloudwatch/README.md)
+
+#### Purpose
+
+Shows how to use the AWS SDK for Python (Boto3) with Amazon CloudWatch to
```

Comment:
Seems a bit redundant. Maybe just "Shows ... to perform the following tasks:" and then list.

Reply:
Agree, fixed.